js/src/features/human-resouce/components/Absence/Absences.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { List } from "antd";
+import { Absence as AbsenceModel } from "HumanResourceModels";
+import { Absence } from "./Abcence";
+
+interface Props {
+  absences: AbsenceModel[];
+  canEdit: boolean;
+  approve: (id: string) => any;
+  remove: (id: string) => any;
+  openUpdateModal: (id: string) => any;
+}
+
+export const Absences: React.FC<Props> = ({
```

Comment:
Again, what is mean `Absences`? Depend on situation. In this case batter `AbsenceList` or similar

Reply:
In what situations Absences mean something different as a list? Usually, plural form means something like a list

Reply:
In our terminology `Absence` is a domain model, `Absences` - type alias to `Absence[]`. And will be great if this models will be the same in every place. But components should be more specific. UI can be different. It like in backend - domain models should have clear names, but for read models you can use postfix `ReadModel`